dev/com.ibm.ws.ejbcontainer.mdb/src/com/ibm/ws/ejbcontainer/mdb/internal/MessageEndpointFactoryImpl.java

```diff
@@ -415,4 +419,71 @@ public String getActivationSpecId() {
     public String getDestinationId() {
         return beanMetaData.ivMessageDestinationJndiName;
     }
+
+    //PausableComponent Methods
+
+    /*
+     * (non-Javadoc)
+     *
+     * @see com.ibm.ws.kernel.launch.service.PauseableComponent#getExtendedInfo()
+     */
```

Comment:
Is there a work item open to track filling in this method?  Javadoc for PausableComponent doesn't explain this well; but at a minimum this should probably include whatever is not included in the name (i.e. if the name is that of the activation spec, then this should include the MDB name, or the other way around.